### PR TITLE
Whole Project

### PR DESCRIPTION
**Pull request by ldelbel**
The objective of this project was to clone newsweek.com using Bootstrap 4.

This project had the goal to improve my understanding of Bootstrap functions and resources, and to help me learn more about how those tools can be used in a webpage and what are the advantages.
In this project I used the following tools from Bootstrap 4:

- Utilities (sizing, positioning, display, color, spacing and flex);
- Components (cards, nav, navbar, forms, buttons);
- Layout (grid, visibility, media);

I kindly ask for review and request allowance to merge.

### CODE REVIEW

**Comment by tirthajyoti-ghosh** [comment]
# Pull Request under review by Tirtha G. (TSE)

Check back within the hour for some feedback 😃

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1924 @@
...
+            </article>
+            <form
+              class="subscribe form-inline d-flex flex-column justify-content-center"
+            >
+              <div
+                class="form-group d-flex align-items-center justify-content-between"
+              >
+                <img class="flex-shrink-0" src="images/news-icon.png" alt="" />
+                <h3 class="darkh3">Start your day with our top 5 articles</h3>
+              </div>
```
Comment (tirthajyoti-ghosh) [inline index.html]: The image and the heading should be beside each other. Currently, they are misaligned.
Annotation 2020-06-04 121048

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1924 @@
...
+                <h3 class="card-title">
+                  Looking for a Powerful, Lightweight and Long-lasting Cordless
+                  Vacuum?
+                </h3>
+                <p class="card-text">
+                  Roborock H6, 2020 Red Dot Award Winner, Ultra-Long lasting
+                  LiPo battery, available now on Amazon.
+                </p>
+              </div>
+            </article>
```
Comment (tirthajyoti-ghosh) [inline index.html]: The top of the image should stick to the top
Annotation 2020-06-04 121601

Remove the extra padding for the other three cards as well

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1924 @@
...
+              <h4>TOP STORY</h4>
+            </div>
+            <article class="card no-border">
+              <div class="img">
+                <a href="#"
+                  ><img
+                    class="card-img-bottom"
+                    src="images/topstory.jpg"
+                    alt="Card image"
+                    style="width: 100%;"
```
Comment (tirthajyoti-ghosh) [inline index.html]: Remove inline styling.

This and in all other places.

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Great job! @ldelbel 🙌🙌

- Awesome README👏. However, your live demo link is not working.
- Your `header` and `main` elements should have a container class to add padding/margin on both sides of the content
Annotation 2020-06-04 120139
- The magazine in the 'In the magazine' section should maintain the same height as other elements when resizing.
Annotation 2020-06-04 122008
- The social media icons in the footer should be beside the Newsweek logo in desktop mode.
- The language section in the footer is misaligned.
Annotation 2020-06-04 122443
- There should be no styles inside HTML files. Remove inline styling
- Remove `!important` from style.css.
- [OPTIONAL] Good job on adding an actual collapsible navbar 👌, however, it is not a hard requirement. I noticed a snap when the hamburger menu is clicked. 
- [OPTIONAL] Copy the image address from the original site so that you don't have to host the images yourself
- [OPTIONAL] Just one card is fine for the sponsored section. just for aesthetics.

I have added some comments under specific lines. Be sure to check them.
Apply the changes and request for another review.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1848 @@
...
+              aria-expanded="false"
+              aria-label="Toggle navigation"
+            >
+              <span class="fas fa-bars"></span>
+            </button>
+          </nav>
+        </div>
+      </div>
+      <nav class="navbot nav position-relative d-none d-lg-flex">
+        <ul class="nav align-items-center">
```
Comment (lucasmazo32) [inline index.html:71]: - [ ] Kindly fix this section for medium and small devices. It should not be displayed.

Source | Clone
-- | --
image | image

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,763 @@
+body,
```
Comment (lucasmazo32) [inline styles.css:1]: - [ ] Be careful with the amount of important that you are using on this file. I would recommend you to create your own classes instead of modifying the ones that boostrap has. Kindly work on that as suggested by the previous code reviewer.

File: backup.html [hunk 6]
```diff
@@ -0,0 +1,895 @@
+<!DOCTYPE html>
```
Comment (lucasmazo32) [inline backup.html]: - [ ] Backup files that you have, or files that are not related to the page should be ignored. Kindly create a file called .gitignore and type in the first line `backup.html`. This will not save the changes in a remote branch but you will have them locally on your machine.

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

For the project to get approved, kindly implement the changes mentioned below.

### Happy coding! :computer:

## Lucas Mazo

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @ldelbel    :handshake: 

### You did a wonderful job :clap:  your project is looking good :medal_sports: and you implemented the changes required by the previous TSEs, nice job. 

### However, there are still some changes to get the project approved :point_down: :
- I see that you added stickler configuration files, but I don't see any stickler checks on pull request, please go to https://stickler-ci.com/ and enable stickler on this repository.
your project: 
Screenshot from 2020-06-05 12-36-24
- what it should be showing:
Screenshot from 2020-06-05 12-37-43

**Comment by ldelbel** [comment]
It's done Haddad. Could you verify again please? I want to approve this
project as soon as possible.

*Lucas Delbel*
Sócio e Co-Fundador
Fitway Doces Fitness
CNPJ: 33.485.118/0001-06
*C +55 35 9 9979-8438*
https://www.instagram.com/fitwaydoces/





Em sex., 5 de jun. de 2020 às 08:38, Sohaib Haddad <notifications@github.com>
escreveu:

**Review by peterrobert — APPROVED** [review]
### APPROVED :clap: :clap: 
**VERY GOOD WORK ** :muscle: 
**YOUR PROJECT LOOK GOOD AND HAVE MADE ALL THE CHANGES REQUESTED** :star: 

ON TO THE NEXT PROJECT AND **HAPPY CODING**
